packages/components/navbar/src/NavbarAccount/NavbarAccount.styles.ts

```diff
@@ -0,0 +1,44 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+import { NavbarAccountProps } from './NavbarAccount';
+
+export const getNavbarAccountStyles = () => ({
+  root: css({
+    // default button reset styles
+    margin: 0,
+    padding: 0,
+    background: 'none',
+    border: 'none',
```

Comment:
```suggestion
    border: 'none',
    appearance: 'none',
```

I'm wondering if other styles are needed here, different browsers usually require different styles for resetting buttons 😭

Reply:
I've copy pasted it from our button tbh, and for some reason, we don't have `appearance: 'none'` there. Do you think we should add it here even though we use our `Button` without resetting it for a while?